src/developer/core/recording-format.md

```diff
@@ -62,6 +62,14 @@ And their corresponding timestamp files should follow the pattern:
 
 - `world_timestamps.npy`, `world_001_timestamps.npy`, `world_002_timestamps.npy`, ...
 
+### Video Player Compatibility
+
+In order to open the video files with the majority of video players, you will have to
+**transcode the video stream encoding from mjpeg to h264** and **change the color format
+from yuvj422p to yuv420p**. You can use the following [ffmpeg](https://ffmpeg.org/)
+command to perform this transcoding:
+> `ffmpeg -i world.mp4 -vf format=yuv420p world_compatible.mp4`
```

Comment:
Do we need `>`?

Reply:
I wanted to be sure that it is being rendered into a newline. Not sure if `>` is required for that. I will give it a try locally.

Reply:
Without `>`
![Screenshot 2020-12-09 at 11 27 25](https://user-images.githubusercontent.com/168390/101617993-aad6f000-3a11-11eb-90f3-69f2dc91f3cc.png)

With `>`
![Screenshot 2020-12-09 at 11 27 35](https://user-images.githubusercontent.com/168390/101617996-ab6f8680-3a11-11eb-800a-72a9d2820dab.png)

@willpatera Please let me know which one you favor. I am ok with both.

Reply:
@papr use 

triple backticks to make a code block

https://github.com/pupil-labs/pupil-docs#code-blocks

Reply:
Done.